package.json

```diff
@@ -109,6 +110,8 @@
     "webpack-cli": "^3.3.11"
   },
   "dependencies": {
+    "@jnoodle/rc-picker": "^1.15.3",
+    "moment": "^2.27.0",
```

Comment:
Is it possible to use the Moment type from rc-picker instead of specify it as a dependency? We can specify it as a dev dependency for usage in tests and doc files.

Reply:
using `dayjs`